contracts/axelar-operators/src/test.rs

```diff
@@ -36,21 +40,40 @@ fn setup_env<'a>() -> (Env, Address, AxelarOperatorsClient<'a>, Address) {
 
 #[test]
 fn test_initialize() {
-    let (env, _, client, _) = setup_env();
+    let (env, contract_id, client, _) = setup_env();
     let user = Address::generate(&env);
 
     client.initialize(&user);
+
+    assert!(env.as_contract(&contract_id, || {
+        env.storage()
```

Comment:
do you mean this: https://docs.rs/assert_ok/latest/assert_ok/ ? feels a little overkill to add a dependency for something like this. 

I added this test to make sure that `initialize()` is impacting the contract's state as intended. If we're just going to check that the function didn't error then it'd probably make more sense to remove it since all the other functions run run `initialize()` too?

Reply:
decided to use assert_ok (can copy or re-export it from soroban-std) when the test is meant to check if the given method didn't error. We can keep using .unwrap for helpers or methods that already have their own dedicated tests